CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 4.6.0 / 2023-07-26
+
+### ✨ New Features
+
+- Add command to really delete data in tracimcli
```

Comment:
```suggestion
- Add command to really delete data with tracimcli command
```
Something like that it is not better?

Do you have an other idea @Quiwy @lebouquetin